component/service/pom.xml

```diff
@@ -201,6 +201,12 @@
       <artifactId>javax.servlet-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <!-- Test -->
+    <dependency>
```

Comment:
for integration social tests

Reply:
the dependency added in commons is in provided scope, this is needed for useRestResourcesTest when testing resize avatars it will need this dependency in test scope or we will have a MethodDefNotFoundException when it calls scaleImage in testes